src/components/StatedBalanceForm.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import BalanceForm from "./BalanceForm";
+
+
+type Props = { getValues: () => { date: Date, value: number }[], saveValues: (values: { date: Date, value: number }[]) => void };
+type State = { value: number };
+
+
+class StatedBalanceForm extends React.Component<Props, State> {
+    save_to_memory(value_from_balanceform: number): void {
+        var values_dates_from_memory = this.props.getValues();
```

Comment:
uno:
use camelCase

duo:
using var is forbidden in our code.
Use const here, or let somewhere else.

Reply:
K, FORBIDDEN. I will remember